### PR TITLE
changed plot to red circles

### PR DESCRIPTION
**Pull request by amantaya**
I prefer red circles for my plots.

### CODE REVIEW

File: merge-conflict-example.R [hunk 1]
```diff
@@ -13,7 +13,7 @@ petal_width <- iris$Petal.Width
 plot(petal_length, petal_width)
 
 # simple plot of petal width
-plot(petal_length, petal_width, pch = 2, col = "purple")
+plot(petal_length, petal_width, pch = 1, col = "red")
```
Comment (Aariq) [inline merge-conflict-example.R:16]: I love triangles! 
```suggestion
plot(petal_length, petal_width, pch = 2, col = "red")
```

**Comment by kepplerm** [comment]
I'm wondering if red would seem a bit aggressive. Maybe blue would be better?

**Review by lossanna — APPROVED** [review]
Triangles are super great

**Review by matiscke — CHANGES_REQUESTED** [review]
triangles look fine, but I think red ones are too aggressive. How about blue triangles?

**Review by linyiyi — COMMENTED** [review]
changes reviewed

**Review by BailieWynbelt — APPROVED** [review]
The triangles look great!

**Pull request merged**